examples/modules/toolkit/plot_postprocessing.py

```diff
@@ -0,0 +1,267 @@
+"""
+
+Postprocessing Tutorial
+=======================
```

Comment:
@alejoe91 : for now this NB is a copy/paste but it need to change because APi have changes and it will overlap with curation and/or qulity metrics.
No ?